verible/verilog/analysis/descriptions.h

```diff
@@ -35,14 +35,16 @@ struct LintConfigParameterDescriptor {
   std::string description;
 };
 
-struct LintRuleDescriptor {
-  LintRuleId name;          // ID/name of the rule.
-  absl::string_view topic;  // section in style-guide
-  std::string desc;         // Detailed description.
-  std::vector<LintConfigParameterDescriptor> param;
-};
-
-}  // namespace analysis
-}  // namespace verilog
-
-#endif  // VERIBLE_VERILOG_ANALYSIS_DESCRIPTIONS_H_
+std::string format_long_description(const std::string& description) {
+    const size_t max_length = 80;
```

Comment:
The indentation is off here, can you run clang-format on your changed files ?
(I think we're using version 17.0.6 it that makes a difference)